### PR TITLE
SPORTS-46 Implement swagger docs

### PR DESCRIPTION
**Pull request by Ishanliv99**
- Implement `Swagger-UI` with `swagger-js-docs`.
- Swagger UI can be found at `/api-docs` from backend.
- `swagger.json` can be found at `/swagger.json` from backend. This is used by Swagger UI.

_NOTE: Run command `npm install` after merging this task._

### CODE REVIEW

File: backend/src/controllers/apiController.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { Request, Response } from 'express';
+import swaggerSpec from '../utils/swagger';
```
Comment (aviskarkc10) [inline backend/src/controllers/apiController.ts]: follow length rule

File: backend/src/index.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ import * as morgan from 'morgan';
 import * as express from 'express';
 import * as bodyParser from 'body-parser';
 
+import path = require('path');
```
Comment (aviskarkc10) [inline backend/src/index.ts]: did you test with import * as path from 'path'?
Also add a comment if import * doesn't work

File: backend/src/routes/index.ts [hunk 3]
```diff
@@ -2,11 +2,19 @@ import { Router } from 'express';
 
 import v1Routes from './v1';
 
-import * as homeController from '../controllers/homeController';
+import * as apiController from '../controllers/apiController';
 
 const router: Router = Router();
 
-router.get('/', homeController.index);
+router.get('/', (req, res) => {
```
Comment (aviskarkc10) [inline backend/src/routes/index.ts]: You haven't described this api

File: backend/src/utils/swagger.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
+// exception : For this package, always extract in this way
+import swaggerJSDoc = require('swagger-jsdoc');
+
+/**
+ * Swagger definition.
+ */
+const definition = {
+  info: {
+    title: "SPORTS",
```
Comment (aviskarkc10) [inline backend/src/utils/swagger.ts]: We should take this from constants

File: backend/src/index.ts (hunk 2 above)
Comment (Ishanliv99) [reply backend/src/index.ts]: It does work
Will push the changes soon

File: backend/.env.example [hunk 5]
```diff
@@ -10,10 +14,10 @@ DB_USERNAME=
...
-LOGGING_DIR=logs
-LOGGING_LEVEL=debug
+LOGGING_DIR=
+LOGGING_LEVEL=
 
 # Tests
 TEST_DB_NAME=
 TEST_DB_USERNAME=
-TEST_DB_PASSWORD=
+TEST_DB_PASSWORD=
```
Comment (sanjeevkpandit) [inline backend/.env.example]: New line missing.

File: backend/src/routes/index.ts (hunk 3 above)
Comment (Ishanliv99) [reply backend/src/routes/index.ts]: Done in swagger-UI

**Pull request merged**